doc/en/user/source/services/wms/get_legend_graphic/index.rst

```diff
@@ -94,6 +94,7 @@ Here is a description of the various parameters that can be used in ``LEGEND_OPT
     - **rows** enables **multirow** layout when layout is **horizontal**. The value is the the maximum rows number of legend. The columns used are equal to the next greater integer of <total of icons>/<number of rows>.
     - **grouplayout** Orientation of groups of layer, possible values are **horizontal** and **vertical** (default if not specified).
     - **countMatched** When set to true, adds at the end of each label the number of features matching that rule in the current map. Requires extra parameters, see details in the :ref:`dedicated section <content-dependent>`.
+    - **hideEmptyRules** When set to true and **countMatched** is also true, hides rules that are not matching any feature. 
```

Comment:
I believe it would be prefereable to have them independent. Something like: 

```suggestion
    + **hideEmptyRules** When set to true hides rules that are not matching any feature. 
```

If one wants both it's easy to add both parameters, the common behavior found in other software is to just hide without count (unfortunately last round of founding was instead about counting without hiding).